pyaedt/application/Design.py

```diff
@@ -516,8 +516,9 @@ def project_properties(self):
             try:
                 settings._project_properties[os.path.normpath(self.project_file)] = load_entire_aedt_file(file_path)
             except Exception:
-                pass
-            self._logger.info("aedt file load time {}".format(time.time() - start))
+                self._logger.info("Failing in loading AEDT file.")
+            else:
+                self._logger.info("aedt file load time {}".format(time.time() - start))
```

Comment:
```suggestion
                self._logger.info("AEDT file load time {}".format(time.time() - start))
```
Make this a complete sentence with a verb and concluding punctuation. I/m not sure how the current message renders.